publish-token/src/main/java/fi/thl/covid19/publishtoken/generation/v1/PublishTokenGenerationRequestArgumentResolver.java

```diff
@@ -0,0 +1,60 @@
+package fi.thl.covid19.publishtoken.generation.v1;
+
+import com.fasterxml.jackson.databind.InjectableValues;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import fi.thl.covid19.publishtoken.Validation;
+import fi.thl.covid19.publishtoken.error.InputValidationException;
+import fi.thl.covid19.publishtoken.error.InputValidationValidateOnlyException;
+import org.springframework.core.MethodParameter;
+import org.springframework.util.FileCopyUtils;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class PublishTokenGenerationRequestArgumentResolver implements HandlerMethodArgumentResolver {
+
+    public static final String VALIDATE_ONLY_HEADER = "X-Validate-Only";
```

Comment:
We've used the prefix KV- for our other headers. The practice of using X- for custom headers is no-longer "the recommended standard" in general, though of course not forbidden either. I do agree that it's good to separate our own koronavilkku stuff with some prefix, which is how we ended up with KV-.

Reply:
Yep, good enough. I just wanted to separate actual client headers from others in some manner, but maybe it's really doesn't matter.